_episodes/08-bonus-episode.md

```diff
@@ -112,10 +111,10 @@ jobs:
       - name: Login and Deploy Container
         run: |
            echo ${{ secrets.GITHUB_TOKEN }} | singularity remote login -u ${{ secrets.GHCR_USERNAME }} --password-stdin oras://ghcr.io
-           singularity push container.sif oras://ghcr.io/${GITHUB_REPOSITORY}:${tag}
+           singularity push container.sif oras://ghcr.io/${GITHUB_REPOSITORY}:${tag
```

Comment:
This change is not introducing a bug with a missing `}` ?

Reply:
I have updated the missing }